Cr

camilo-burbano opened the pull request.

J4ckDev committed:
Carpeta donde estarán los archivos del XBlock y archivo readme para el XBlock generado

J4ckDev committed:
Carpeta donde estarán los archivos del XBlock y el archivo readme para el XBlock generado

LeandroPotes committed:
Add files via upload

LeandroPotes committed:
Update student_adaptive_test.js

Correcciones en el HTML de la función loadHoneyAlonso realizadas.

camilo-burbano committed:
Add files via upload

camilo-burbano committed:
Delete myxblock.css

camilo-burbano committed:
Delete myxblock.html

camilo-burbano committed:
Delete myxblock.js

camilo-burbano committed:
Delete myxblock.py

camilo-burbano committed:
Delete studio_view.css

camilo-burbano committed:
Delete studio_view.html

camilo-burbano committed:
Delete studio_view.js

camilo-burbano committed:
Add files via upload

camilo-burbano committed:
Delete studio_view.js

camilo-burbano committed:
Add files via upload

LeandroPotes committed:
Add files via upload

dsrestrepo commented on CR/myxblock/myxblock/myxblock.py:
```diff
@@ -0,0 +1,103 @@
+        
+        conn = psycopg2.connect(database='GraficarDatos',user='camilo',password='bea24',host='localhost')
+        cur = conn.cursor()
+        cur.execute("SELECT valor FROM graficar")
```
Aquí lo único seria cambiar el nombre de **valor** de su base de datos a **resultado** para tener todos la misma base de datos; los campos acordados fueron: Id_curso:Integer, Id_estudiante:Integer[PK], fecha:date, nombre_test: character varying(30), resultado:character varying(30)

J4ckDev commented on CR/myxblock/myxblock/static/js/src/studio_view.js:
```diff
@@ -0,0 +1,98 @@
+    
+    z = a + b + c + d;
+
+    o = Math.floor(((a * 100) / z) * 100) / 100;
```
Generalizar para todos los estilos de aprendizaje

J4ckDev commented on CR/myxblock/myxblock/static/js/src/studio_view.js:
```diff
@@ -0,0 +1,98 @@
+      }
+    }
+
+    var grafica = document.getElementById('grafico').getContext('2d');
```
Quitar el diagrama de pastel y solo dejar el de barras

J4ckDev reviewed (changes requested):
Además de la notación de David, arreglar estos nuevos cambios

LeandroPotes committed:
Cambios con id_usuario y vista del profesor

Se suben a la base de datos el id_usuario obtenido por el xblock, se puede comparar si cada usuario ha presentado o no los respectivos test para mostrar el resultado o el test en caso contrario.
Se modificó el studio_view añadiendo una vista(studio_analytics) para permitir que el docente mire los resultados de los tests y pueda volver a elegir otro test.

LeandroPotes committed:
almacenar datos en variables de xblock

Se comenta el código que permitía subir los datos a una base local, para ahora almacenarlos en las variables globales del xblock, las cuales se encuentran en el archivo .py. Además se hacen algunas correciones en el código para que el funcionamiento de la plataforma no se altere al hacer estos cambios.

J4ckDev committed:
Merge branch 'tea' into cr

Kalevins committed:
Add files via upload

Primera versión de integración

Kalevins committed:
Create README.md
